test/types.go

```diff
@@ -19,6 +20,23 @@ type Input struct {
 	RAWRequest     string         `yaml:"raw_request,omitempty" koanf:"raw_request,omitempty"`
 }
 
+// Overrides represents the overridden inputs that have to be applied to tests
+type Overrides struct {
```

Comment:
Little attempt to move the overrides to a new struct. When it will be settled, I will open another PR for extending the override capabilities (all the ones that I have commented out are currently not enforced)